custom-recipes/ab-testing-split/recipe.json

```diff
@@ -0,0 +1,139 @@
+{
+    "meta": {
+        "label": "AB testing split",
+        "description": "Split a list of users into a A group and a B group.",
+        "icon": "icon-beaker"
+    },
+    "selectableFromDataset": "user_list",
+    "selectableFromFolder": "folder",
+
+    "kind": "PYTHON",
+
+    "inputRoles": [
+        {
+            "name": "user_list",
+            "label": "Dataset with the reference of the users involved in the experiment ",
+            "description": "Dataframe containing the ids, the emails of the users in the experiment",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": true
+        },
+        {
+            "name": "folder",
+            "label": "Folder",
+            "description": "Folder containing the parameters",
+            "arity": "UNARY",
+            "required": false,
+            "acceptsDataset": false,
+            "acceptsManagedFolder": true
+        }
+    ],
+    "outputRoles": [
+        {
+            "name": "groups",
+            "label": "Experiment dataset",
+            "description": "Input dataset with an extra column containing the group",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": true
+        },
+        {
+            "name": "A_group",
+            "label": "A group (optional)",
```

Comment:
You have all the necessary information in the "groups" dataset (label = experiment dataset), since you know which email belongs to which group. I thought it would be more convenient for some final users to have 2 separate datasets though. But as it was not necessary, Léo advised to make them optional.